TheEthicalHackerCup/Assets/DefenseMenu/Scripts/AttackAttemptText.cs

```diff
@@ -35,6 +32,16 @@ void Update()
         green = new Color(0, 189f/255f, 63f/255f);
     }
 
+    // Update is called once per frame
+    void Update()
+    {
+        attemptCompleted = GameManager.Instance.GetAttackMinigamesAttempted(sc);
+        attemptsTotal = GameManager.Instance.GetDefenseUpgradeLevels(sc) + 1;
+        string attempts = attackAttemptText + "\n" + attemptCompleted + "/" + attemptsTotal;
```

Comment:
Polling this every frame might cause issues for performance.
I feel like this can get setup when this scene loads. The atk_minigame attempts and defense upgrade process will only be changed when the player is in a different scene.

Also incoming update from me: Adding a class that stores attackAttempts, currentUpgradeLevel, maxUpgradeLevel

Reply:
moved it away to Start function